.github/workflows/install-test.yml

```diff
@@ -0,0 +1,45 @@
+on:
+  schedule:
+    - cron: '0 0 * * *' # Every day at midnight
+  push:
+    tags:
+    - '*'
+name: Package manager test
+jobs:
+  homebrew:
+    runs-on: macos-latest
+    steps:
+      - run: brew install stripe/stripe-cli/stripe
+      - run: stripe --version
+  apt:
+    runs-on: ubuntu-latest
+    steps:
+      - run: curl -s https://packages.stripe.dev/api/security/keypair/stripe-cli-gpg/public | gpg --dearmor | sudo tee /usr/share/keyrings/stripe.gpg
+      - run: echo "deb [signed-by=/usr/share/keyrings/stripe.gpg] https://packages.stripe.dev/stripe-cli-debian-local stable main" | sudo tee -a /etc/apt/sources.list.d/stripe.list
+      - run: sudo apt update
+      - run: sudo apt install stripe
+      - run: stripe --version
+  yum:
+    runs-on: ubuntu-latest
+    container: ubuntu:18.04
+    steps:
+      - run: apt-get update
+      - run: apt-get -y install yum
```

Comment:
sad we have to do this intead of `runs-on: centos` but i see github actions doesn't support any linux runners besides ubuntu :(

Reply:
Not ideal but it should work in perpetuity without us having to fiddle with it